### ISSUE TITLE
HTTP 403 when launching client

### ISSUE THREAD

**Issue by PaulBenn** [irtimaled/BoundingBoxOutlineReloaded#55]
I am on vanilla Minecraft 1.14. I've downloaded the installer and let it do its thing - then restarted launcher, so that the BBOR profile was created successfully and hit play.

I get the following, in an error pop-up:

```
Name: 1.14.jar
URL: https://s3.amazonaws.com/Minecraft.Download/versions/1.14/1.14.jar
Error details: HTTP 403: Forbidden
Filename on disk: 3b7a-693b-c40f-e5f5
Path: C:\Users\PAULBE~1\AppData\Local\Temp\3b7a-693b-c40f-e5f5
Exists: file
```

Any ideas?

**Comment by leftyt13** [comment]
Same here. Downloaded the 1.14 jar and ran the installer as directed. BBOR profile was added to launcher, but will not play. Here's my error info:

```
Failed to download file. 1.14.jar
Name: 1.14.jar
URL: https://s3.amazonaws.com/Minecraft.Download/versions/1.14/1.14.jar
Error details: HTTP 403: Forbidden
Filename on disk: 2264-bb0a-2e40-c434
Path: C:\Users\teh1l\AppData\Local\Temp\2264-bb0a-2e40-c434
Exists: file
```

**Comment by irtimaled** [comment]
The BBOR profile doesn't directly reference the location of the 1.14.jar file, it instead inherits from the 1.14 profile. The 1.14 profile you have, is it called 1.14?

**Comment by PaulBenn** [comment]
Hi, when I started playing again recently I downloaded 1.14.1 directly, such that the 1.14 JAR was not there. It seems since you inherit from 1.14 instead of the latest version, if you do not have a local 1.14 JAR this error will happen.

I cannot seem to get it working for 1.14.2. Would you be able to make this happen for us please? I took a look at the source code but cannot see where exactly the inheritance is happening.

**Comment by irtimaled** [comment]
You are using the 1.14.2 version of the mod, and also have a profile for 1.14.2 (un-modded) with the name 1.14.2?

**Comment by PaulBenn** [comment]
Was not aware you'd published a 1.14.2 version, but the JAR produced by building your 1.14.2 branch does work with 1.14.2. Cheers Irti
